math/CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ Ref: https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.j
 
 ### Features
 
+* [#18421](https://github.com/cosmos/cosmos-sdk/pull/18421) Add mutative api for `LegacyDec.BigInt()`.
```

Comment:
This should be moved under unreleased features.

Reply:
Ok, I was confused